### ISSUE TITLE
[<Library component: Model|Core|etc...>] AutoModel Hyperparameter input_size_multiplier is wrong

### ISSUE THREAD

**Issue by zllearn** [Nixtla/neuralforecast#1222]
### What happened + What you expected to happen

使用automodel模型时，其default_config提供的参数名input_size_multiplier错误，在实际使用时是input_size

### Versions / Dependencies

neuralforecast库
python3.8 和 python3.10.9

### Reproduction script

    default_config = {
        "input_size_multiplier": [-1, 4, 16, 64],
        "inference_input_size_multiplier": [-1],
        "h": None,
        "encoder_hidden_size": tune.choice([50, 100, 200, 300]),
        "encoder_n_layers": tune.randint(1, 4),
        "context_size": tune.choice([5, 10, 50]),
        "decoder_hidden_size": tune.choice([64, 128, 256, 512]),
        "learning_rate": tune.loguniform(1e-4, 1e-1),
        "max_steps": tune.choice([500, 1000]),
        "batch_size": tune.choice([16, 32]),
        "loss": None,
        "random_seed": tune.randint(1, 20),
    }

### Issue Severity

Medium: It is a significant difficulty but I can work around it.

**Comment by marcopeix** [comment]
Hello! What is wrong exactly with the `input_size_multiplier` and what error do you see?

**Comment by zllearn** [comment]
您好！
我最近较忙没有看见邮箱，不好意思！
我之前反馈的问题是我自己看错了，我在传入AutoTimeMixerd的config参数时，给错了input_size的命名为input_size_multiplier
所以导致了后面的错误，后来改成了input_size就没有问题了
感谢您的回信！


错误参数名：
config_AutoTimeMixer = {
&nbsp; &nbsp; "input_size_multiplier": tune.choice([INPUT_SIZE, INPUT_SIZE*2]),&nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "d_model": tune.choice([32, 64]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "e_layers": tune.choice([2, 4]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
&nbsp; &nbsp; "learning_rate": tune.loguniform(1e-4, 1e-2),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
&nbsp; &nbsp; "batch_size": tune.choice([64, 128]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "max_steps": tune.choice([1500]),
&nbsp; &nbsp; "random_seed": tune.randint(1, 10)
}
&nbsp; &nbsp;&nbsp;
正确参数名：
config_AutoTimeMixer = {
&nbsp; &nbsp; "input_size": tune.choice([INPUT_SIZE, INPUT_SIZE*2]),&nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "d_model": tune.choice([32, 64]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "e_layers": tune.choice([2, 4]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
&nbsp; &nbsp; "learning_rate": tune.loguniform(1e-4, 1e-2),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
&nbsp; &nbsp; "batch_size": tune.choice([64, 128]),&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; "max_steps": tune.choice([1500]),
&nbsp; &nbsp; "random_seed": tune.randint(1, 10)
}
&nbsp; &nbsp;&nbsp;
model = AutoTimeMixer(
&nbsp; &nbsp; h=HORIZON,&nbsp;
&nbsp; &nbsp; n_series=1,&nbsp;
&nbsp; &nbsp; config=config_AutoTimeMixer,&nbsp;
&nbsp; &nbsp; loss=MAE(),&nbsp;
&nbsp; &nbsp; num_samples=20,&nbsp;
&nbsp; &nbsp; verbose=False,
&nbsp; &nbsp; )



张乐
***@***.***



&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "Nixtla/neuralforecast"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月24日(星期二) 凌晨0:18
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Nixtla/neuralforecast] [<Library component: Model|Core|etc...&gt;] AutoModel Hyperparameter input_size_multiplier is wrong (Issue #1222)





 
@zllearn 你有碰到什么报错吗？这是Neuralforecast 里采用的命名而已。实际上还是尝试不同input_size的参数来优化。
